rotkehlchen/accounting/structures/base.py

```diff
@@ -169,26 +169,24 @@ def deserialize_from_db(cls, entry: HISTORY_EVENT_DB_TUPLE_READ) -> 'HistoryBase
     @property
     def serialized_event_identifier(self) -> str:
         """Take a HistoryBaseEntry's event_identifier and returns a string representation."""
-        if self.location == Location.KRAKEN:
+        if self.location == Location.KRAKEN or self.event_identifier.startswith('rotki_events'.encode()):  # noqa: E501
             return self.event_identifier.decode()
 
         hex_representation = self.event_identifier.hex()
         if hex_representation.startswith('0x') is True:
             return hex_representation
         return '0x' + hex_representation
 
-    @classmethod
-    def deserialize_event_identifier_from_kraken(cls, val: str) -> bytes:
-        """Takes a `refid` from Kraken exchange and turns it into a bytes event_identifier."""
-        return val.encode()
-
     @classmethod
     def deserialize_event_identifier(cls, val: str) -> bytes:
```

Comment:
I actually meant to keep the old functions and create the new combined one and use them where needed, but what you did is also fine though it uses one extra if in many cases it  can avoid it.